packages/mapviewer/tests/cypress/support/commands.js

```diff
@@ -81,11 +81,8 @@ Cypress.Commands.add(
             }
         }
 
-        if (!('lang' in queryParams)) {
-            queryParams.lang = 'en'
-        }
```

Comment:
I'd maybe keep that, we can't be certain that the CI instance we receive will have its locale as English, and some tests are testing for translated texts, so we really need to be 100% certain we are using English by default.

Reply:
Ah yes, I forget to put it back. There was an issue with redo the search in local instance but not in the cypress test (since it's always set to 'en').
It got me thinking, it might be not the best way, since the case where the lang is not set in the URL is not covered in our test case. Anyway, perhaps it's good for now.